docs/_get-started/01-overview.md

```diff
@@ -50,7 +50,8 @@ Dart has an excellent package manager called "pub" (<https://pub.dev>).  It is p
 
 ### The Challenge of Justifying Trying a New Language
 
-[This StackOverflow answer](https://stackoverflow.com/questions/53007782/what-benefits-does-chisel-offer-over-classic-hardware-description-languages) about why it's worth trying Chisel (an alternative to ROHD) contains valuable insight into why it is difficult in general to justify a new language to someone who hasn't used it before:
+<!-- markdownlint-disable -->[This StackOverflow answer](https://stackoverflow.com/questions/53007782/what-benefits-does-chisel-offer-over-classic-hardware-description-languages) about why it's worth trying Chisel (an alternative to ROHD) contains valuable insight into why it is difficult in general to justify a new language to someone who hasn't used it before:
```

Comment:
For the stackoverflow links, can you try changing the mlc_config instead?  Like this?
https://github.com/intel/rohd/pull/463/files#diff-661ef3cfb3b5a7424be7e05effe7bc6a2da390ce10999cf0cb6d979c2e906805